packages/cms-cli/lib/validation.js

```diff
@@ -130,7 +133,48 @@ function validateMode(command) {
   return false;
 }
 
+const getAbsoluteFilePath = _path => {
```

Comment:
I still need to move this to a different file, since it isn't validation related

Reply:
I think `cms-lib/path` might be a good place for this.

Reply:
That said, it is a little bit surprising that this is mixing validation with resolving and that it returns `null` if the path is not a file.

Reply:
I can split it into 2 functions. `getAbsolutePath` which will just `return path.resolve(getCwd(), _path)` and `validatePath` as a separate function